DFC.ServiceTaxonomy.ApiFunction/CypherQueries/GetOccupationsByLabel.json

```diff
@@ -1,5 +1,5 @@
 {
-  "query": "with toLower($label) as lowerlabel\nmatch (o:esco__Occupation)\nwhere toLower(head(o.skos__prefLabel)) contains lowerlabel or \ncase toLower($matchaltlabels)\n  when 'true' then\n    any(alt in o.skos__altLabel where toLower(alt) contains lowerlabel)\n    or any(hidden in o.skos__hiddenLabel where toLower(hidden) contains lowerlabel)\n  else\n    false\n  end\nwith { occupations:collect(\n{\n  uri:o.uri,\n  occupation:head(o.skos__prefLabel),\n  alternativeLabels:coalesce(o.skos__altLabel, o.skos__hiddenLabel),\n  lastModified:head(o.dct__modified),\n  matches:\n  {\n    occupation:[preflab in o.skos__prefLabel where toLower(preflab) contains lowerlabel],\n    alternativeLabels:coalesce([altlab in o.skos__altLabel where toLower(altlab) contains lowerlabel],\n      [hidlab in o.skos__hiddenLabel where toLower(hidlab) contains lowerlabel])\n  }\n}\n)} as occupations \nreturn occupations\n",
+  "query": "with toLower($label) as lowerlabel\nmatch (o:esco__Occupation)\nwhere toLower(o.skos__prefLabel) contains lowerlabel or \ncase toLower($matchaltlabels)\n  when 'true' then\n    any(alt in o.skos__altLabel where toLower(alt) contains lowerlabel)\n    or any(hidden in o.skos__hiddenLabel where toLower(hidden) contains lowerlabel)\n  else\n    false\n  end\nwith { occupations:collect(\n{\n  uri:o.uri,\n  occupation:o.skos__prefLabel,\n  alternativeLabels:coalesce(o.skos__altLabel, o.skos__hiddenLabel),\n  lastModified:o.dct__modified,\n  matches:\n  {\n    occupation:[preflab in o.skos__prefLabel where toLower(preflab) contains lowerlabel],\n    alternativeLabels:coalesce([altlab in o.skos__altLabel where toLower(altlab) contains lowerlabel],\n      [hidlab in o.skos__hiddenLabel where toLower(hidlab) contains lowerlabel])\n  }\n}\n)} as occupations \nreturn occupations\n",
```

Comment:
No need to coalesce the hidden labels if there are no alt labels anymore, as that was just to work around the 2, now fixed, occupations with alt labels rather than hiddern labels (`alternativeLabels:coalesce(o.skos__altLabel, o.skos__hiddenLabel)`).